### PR TITLE
Use clipboardjs

### PR DESCRIPTION
**Pull request by root-beer**
Use clipboard.js instead of ZeroClipboard.

https://clipboardjs.com/

### CODE REVIEW

**Comment by root-beer** [comment]
Please Review and Merge PR. @Joeper214 :bow:

File: app/assets/javascripts/share.js [hunk 1]
```diff
@@ -218,3 +218,17 @@ $(document).on("drop", ".allow_textfile_drop", function(e){
   };
   fileReader.readAsText(file);
 });
+
+// setup clipboard.js
+new Clipboard('[data-clipboard]').on('success', function (e) {
```
Comment (Joeper214) [inline app/assets/javascripts/share.js]: :+1:

**Comment by Joeper214** [comment]
I'm getting this following error.


clib_err



clip_err2

**Comment by Joeper214** [comment]
Maybe this needs to be updated from the develop branch?

**Comment by Joeper214** [comment]
Hmmm... I updated it to develop branch. but still error occurs.

**Comment by root-beer** [comment]
Did you try the update process?
```
$ npm i
$ gulp tsd
$ gulp ts
```

**Comment by Joeper214** [comment]
There is a module that is skipped when running `npm i`.

Maybe this only occurs in ubuntu os?


not_compatible

**Comment by Joeper214** [comment]
After Merging. #214     problem is gone.

**Comment by Joeper214** [comment]
LGTM!

**Comment by root-beer** [comment]
:+1:

**Comment by Joeper214** [comment]
Thanks @root-beer

**Comment by root-beer** [comment]
:bow:

**Pull request merged**